source/includes/_cds_services.md

```diff
@@ -250,14 +250,21 @@ Field | Description
 `url` | *URL*. An optional absolute URL to load (via `GET`, in a browser context) when a user clicks on this link to learn more about the organization or data set that provided the information on this card. Note that this URL should not be used to supply a context-specific "drill-down" view of the information on this card. For that, use `link.url` instead.
 `icon` | *URL*. An optional absolute URL to an icon for the source of this card. The icon returned by this URL should be in PNG format, an image size of 100x100 pixels, and must not include any transparent regions.
 
-Each **Suggestion** is described by the following attributes.
+Each **Suggestion** is described by the following attributes. 
 
 Field | Description
 ----- | -----------
 `label` |  *string*. human-readable label to display for this suggestion (e.g. the EHR might render this as the text on a button tied to this suggestion).
 `uuid` | *string*. unique identifier for this suggestion. For details see [Suggestion Tracking Analytics](#analytics)
-<nobr>`create`</nobr> | *string*. new resource(s) that this suggestion applies within the current activity (e.g. for `medication-prescribe`, this holds the updated prescription as proposed by the suggestion).  
-`delete`  | *string*. id(s) of any resources to remove from the current activity (e.g. for the `order-review` activity, this would provide a way to remove orders from the pending list). In activities like `medication-prescribe` where only one "content" resource is ever relevant, this field may be omitted.
+`actions` | *array*. array of objects, each defining a suggested action. Within a suggestion, all actions are logically AND'd together, such that a user selecting a suggestion selects all of the actions within it. Note that suggestions are implicitly OR'd. 
+
+Each **Action** is described by the following attributes.
+
+Field | Description
+----- | -----------
+`type` |  *string*. action's crud type. Allowed values are: `create`, `update`, `delete`. 
+`description` | *string*. human-readable description of the suggested action. May be presented to the end-user. 
+`resource` | *object*. depending upon the `type` attribute, new resource(s) or id(s) of resources. For a type of `create`, the `resource` attribute contains new FHIR resources to apply within the current activity (e.g. for `medication-prescribe`, this holds the updated prescription as proposed by the suggestion).  For `delete`, id(s) of any resources to remove from the current activity (e.g. for the `order-review` activity, this would provide a way to remove orders from the pending list). In activities like `medication-prescribe` where only one "content" resource is ever relevant, this field may be omitted. For `update`, existing resources to modify from the current activity (e.g. for the `order-review` activity, this would provide a way to annotate an order from the pending list with an assessment). This field may be omitted.
```

Comment:
Yes, I agree that this would be useful, and am willing to work on improving our suggestion documentation, including examples, but not as part of this issue.